src/store/index.js

```diff
@@ -5,6 +5,7 @@ import mutations from './mutations';
 import backend from './modules/backend';
 import persistState from './plugins/persistState';
 import modals from './plugins/modals';
+/* eslint import/no-cycle: [2, { maxDepth: 1 }] */
```

Comment:
we should avoid cyclic dependencies

Reply:
I agree. To avoid this, most of the current implementation needs to be changed.

Reply:
yes, that would be necessary